step.yml

```diff
@@ -26,8 +26,13 @@ inputs:
   - plist_path:
     opts:
       title: "Info.plist file path"
+      summary: Path to the given target's Info.plist file.
       description: |
-        Path to the given project's Info.plist file.
+        **NOTE:**<br/>
```

Comment:
could you please post here a screenshot of this input in a closed and an opened state from workflow editor?

Reply:
<img width="1552" alt="Screenshot 2019-05-13 at 14 53 46" src="https://user-images.githubusercontent.com/37296013/57622946-fbb10900-758e-11e9-8ffb-426c57fce419.png">
<img width="1552" alt="Screenshot 2019-05-13 at 14 53 52" src="https://user-images.githubusercontent.com/37296013/57622953-ff449000-758e-11e9-9912-3708035fb16e.png">

Reply:
I think this way it is not very straightforward for the users. They will see the description only if they expand the input field. Also there is a typo: `on by one`

Reply:
Fixed